addons/google_calendar/models/calendar_recurrence_rule.py

```diff
@@ -40,7 +40,7 @@ def _apply_recurrence(self, specific_values_creation=None, no_send_edit=False, g
                     'active': False,
                     'need_sync': True,
                 }]
-                event._google_delete(google_service, event.google_id)
+                event.with_context(send_updates=False)._google_delete(google_service, event.google_id)
```

Comment:
There is a risk to miss some emails notification but missing one is better than sending too many.
I is always possible to check the event in the calendar or to confirm it.
New events should send a modification for the new recurrence right ?

Reply:
Correct, it is being sent 🙂